resources/views/admin/imet/v2/modules_iterate.blade.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/** @var array $modules */
+/** @var array $imet_keys */
+/** @var array $imet_eval_keys */
+/** @var array $countries */
+/** @var array $years */
+/** @var array $wdpa */
+/** @var object $request */
+/** @var string $method */
+/** @var string $url */
+/** @var string $results */
+
+?>
+
+@extends('layouts.admin')
+
+@section('content')
+<div class="module-container" id="table_list">
+    <div class="module-body">
+        <form  method="{{ $method }}" action="{{URL::route('csv_list')}}">
+            {{ csrf_field() }}
+            <div >
+                <table class="table module-table">
+                    <tr id="imet_details">
+                        <td class="align-baseline text-center">
+                            {!! \App\Library\Ofac\Input\Input::label('country', trans('entities.common.country')) !!}
+                            {!! \App\Library\Ofac\Input\DropDown::simple('country', $request->input('country'), $countries) !!}
+                        </td>
+                        <td class="align-baseline text-center">
+                            {!! \App\Library\Ofac\Input\Input::label('year', trans('entities.common.year')) !!}
+                            {!! \App\Library\Ofac\Input\DropDown::simple('year', $request->input('year'), $years) !!}
+                        </td>
+                        <td  class="align-baseline text-center">
+                            {!! \App\Library\Ofac\Input\Input::label('year', trans('entities.common.protected_area')) !!}
+                            {!! \App\Library\Ofac\Input\DropDown::simple('wdpa', $request->input('wdpa'), $wdpa) !!}
+                        </td>
+                        <td  class="align-baseline text-center">
+                           <br/>
+                            <button type="submit" class="btn btn-nav rounded mt-2">{{ ucfirst(trans('common.apply_filters')) }}</button>
+                        </td>
+                    </tr>
+                </table>
+            </div>
+
+        </form>
+    </div>
+</div>
+    @if(Str::length($results) > 0)
+        @foreach($modules as $key => $module)
```

Comment:
There's a mismatch on the way you gave names to variables:
- in line 49 $key => $module should be $step_key => $step
- in line 68 @foreach($module as $ind_key => $indicator) should be @foreach(steps $module_key => $module)